examples/WiFiAdvancedChatServer/WiFiAdvancedChatServer.ino

```diff
@@ -0,0 +1,117 @@
+/*
+ Advanced WiFi Chat Server
+
+ A more advanced server that distributes any incoming messages
+ to all connected clients but the client the message comes from.
+ To use, telnet to your device's IP address and type.
+ You can see the client's input in the serial monitor as well.
+
+ Circuit:
+  * Board with NINA module (Arduino MKR WiFi 1010, MKR VIDOR 4000 and UNO WiFi Rev.2)
+
+ created 18 Dec 2009
+ by David A. Mellis
+ modified 9 Apr 2012
+ by Tom Igoe
+ redesigned to make use of operator== 25 Nov 2013
+ by Norbert Truchsess
+ changed for server.accept() in July 2018
+ by Paul Stoffregen for Ethernet library
```

Comment:
```suggestion
```
Although Paul Stoffregen very much deserves recognition for this contribution (https://github.com/arduino-libraries/Ethernet/commit/2304c69e08450ce509fdcca5a01f6a514136a93c), My opinion is that the practice of adding a changelog to the example sketch comments is harmful (https://github.com/arduino/Arduino/issues/7830). This information is not of value or interest to most users and the commit history will generally be much more useful to the rest. It will inevitably grow over time, making the example less and less approachable to newcomers as they must scroll through a screen of cryptic text before even reaching the code.

The removal of existing changelogs (e.g., the lines above) is maybe more challenging, but Paul Stoffregen did not add an entry in https://github.com/arduino-libraries/Ethernet/commit/2304c69e08450ce509fdcca5a01f6a514136a93c so this is the introduction of it.